find_target_name_from_header.py

```diff
@@ -137,34 +107,42 @@ def query_simbad(date,coords,name=None,limit_G_mag=15,metadata=None,verbose=True
 
     if name is not None:
         search = customSimbad.query_object(name)
-        validSearch = search[search['FLUX_G']<limit_G_mag]
-        nb_stars = len(validSearch)                
+        
+        if type(search)==type(None): # a binary system component may not be identified by Simbad()
```

Comment:
Thanks for fixing this problem I had not noticed about binaries !

Reply:
You're welcome :)